### DISCUSSION TITLE
CUSTOMIZATION

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#144] [General]
### This thread is dedicated to all sorts of customization in the Georgia-ReBORN theme.
**You can post scripts or code and all kinds of other stuff you can think of.
[Basic customization can be found in the FAQ here under section `CUSTOMIZATION`](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/58)**

---

<details>
<summary>How to remove the two digit country ISO code from the Catalog entry in the metadata grid in Details?</summary>

Open your `georgia-reborn-config.jsonc` and go to the `metadataGrid:` section.<br />

**Replace original Catalog pattern:**
`{ "label": "Catalog", "val": "$puts(cn,$if3(%catalognumber%,%discogs_catalog%,))[$if($get(cn),$get(cn)[ / $replace($if3(%releasecountry%,%discogs_country%,),AF,XW)],)]" },`

**with this:**
`{ "label": "Catalog", "val": "$puts(cn,$if3(%catalognumber%,%discogs_catalog%,))[$if($get(cn),$get(cn)[ $replace($if3(%releasecountry%,%discogs_country%,),$if3(%releasecountry%,%discogs_country%,),,)],)]" },`
</details>

<details>
<summary>How to add channels entry in the metadata grid in Details?</summary>

Open your `georgia-reborn-config.jsonc` and go to the `metadataGrid:` section and add new entry:<br />
`{ "label": "Channels", "val": "$caps([%channels%])" },`
</details>

---

**Comment by d3rb** [comment]
Hi Tom,
i played a little bit with photoshop and have a idea.
What do you think about this view. I don't know if it's posible?

greetings
Enrico

![foobar2000 new view](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/0cdacace-f0ed-47eb-a741-b1a74ae27d6b)

**Reply by TT-ReBORN** [reply]
Hi Enrico,

this won't work for proportional player sizes as the right width space is too small for a playlist, so the practical use isn't good. 
Also adding new layout views is really complex and a lot of work because the theme is using one Spider Monkey Panel without any containers. But it could be cool to display the album tracks in the cover, like when you see the back cover where all songs of the albums are listed. It would work like when you right click on the album art > Display lyrics but instead Display album tracks.

I will keep this idea in mind for the future but first I need to release v3.0 final and before that, release RC-2 
and make the Georgia-ReBORN website with full documentation.

-Tom

**Comment by silverwings15** [comment]
@TT-ReBORN can you please repost the code for hiding menu items? i wanna hide some tabs like Rating and Bio

**Reply by TT-ReBORN** [reply]
You mean the top menu buttons? 
Top menu `Options` > `Player controls` > `Top menu` > `Default` or `Artwork`.
Then you can choose which panel menu buttons you want to hide.

**Reply by silverwings15** [reply]
ah yes thank you, i missed that this was incorporated into the settings

**Comment by d3rb** [comment]
Hello Tom need help, which file do I have to edit to change the menu colors?

![menü](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/345be119-30f2-4f08-83d1-87e72f47a14f)

**Reply by TT-ReBORN** [reply]
Hi Enrico,

unfortunately this is not possible because it is controlled by foobar -> Windows shell. 
The menu's have either white or dark color ( if dark mode is enabled ).

-Tom
